DirectEmployers/MyJobs
# [PD-3275] Editing the table presentation

## What's this PR do?
Edits the table presentation

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of any of the analytics reports

## How should this be manually tested?
Pull Down repository. Check the bottom of the table presentation at all of the analytic reports to make sure it isn't overflowing

## Any background context you want to provide?
Needed to fix the presentation at the bottom of site traffic report
